app/controllers/application_controller.rb

```diff
@@ -137,4 +139,10 @@ def theme_chosen
     end
   end
 
+  protected
+
+  def configure_permitted_parameters
```

Comment:
Does this have to live in `application_controller` or can it move to a child controller elsewhere?